### DISCUSSION TITLE
Does trino have plans to support InfluxDB as a plugin connector?

### DISCUSSION THREAD

**Discussion by BigStupideer** [trinodb/trino#11812] [General]
as topic

**Comment by findepi** [comment]
See https://github.com/trinodb/trino/pull/2397

**Reply by BigStupideer** [reply]
#2397 seems has been stoped two years ago, a lot of methods and interfaces have changed since that PR and some problems found in code review still left.
#8421 also mentioned to work on influxdb connector plugin but no following now.
i'd like to start working on it from scratch, make a minimal PR then iterate on it over time. is that OK？

**Reply by ebyhr** [reply]
@BigStupideer Could you ask in #8421?
